lecture_01/requirements.txt

```diff
@@ -0,0 +1,2 @@
+black
```

Comment:
I have noticed that from version to version black may change some formatting.  Should we fix the version?

Reply:
yes, I'd pin the latest